.github/actions/trivy/action.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Trivy Action'
+description: 'Docker Image Trivy security check combined action'
+
+inputs:
+  image-tags:
+    required: true
+    type: string
+  image-name:
+    required: true
+    type: string
+
+runs:
+  using: "composite"
+  steps:
+    - name: Set variables
+      env:
+        TAGS: ${{ inputs.image-tags }}
+        NAME: ${{ inputs.image-name }}
+      id: vars
+      shell: bash
+      run: |
+        tag=$(echo "$TAGS" | head -1)
+        name=${NAME// /-}
+        echo "::set-output name=tag::${tag}"
+        echo "::set-output name=name::${name}"
+
+    - name: Run Trivy vulnerability scanner on ${{ inputs.image-name }} image
+      uses: aquasecurity/trivy-action@master
```

Comment:
do they really recommend using master? or should it be some kind of version tag?

Reply:
Good question. This is based on their documentation. I guess we want to use the latest version always.